### PR TITLE
0002 enhance disk sort

### PR DESCRIPTION
**Pull request by ycheng**
fix issue #109 and #110.

### CODE REVIEW

File: ubiquity/stepDellBootstrap.ui [hunk 1]
```diff
@@ -616,6 +616,30 @@
...
+                  <object class="GtkAlignment" id="alignment13">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="left_padding">25</property>
+                    <child>
+                      <object class="GtkLabel" id="raid_warning">
+                        <property name="width_request">600</property>
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+			<property name="label" translatable="yes">Disk with * in the front is part of a raid.</property>
```
Comment (superm1) [inline ubiquity/stepDellBootstrap.ui:629]: Slight wording tweak: `Are part of a RAID array`.

File: ubiquity/stepDellBootstrap.ui [hunk 2]
```diff
@@ -616,6 +616,30 @@
...
                 </child>
+                <child>
+                  <object class="GtkAlignment" id="alignment13">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="left_padding">25</property>
+                    <child>
+                      <object class="GtkLabel" id="raid_warning">
+                        <property name="width_request">600</property>
+                        <property name="visible">True</property>
```
Comment (superm1) [inline ubiquity/stepDellBootstrap.ui:627]: Can you hide this label by default, and add only show this message when a RAID array has been detected?

File: ubiquity/stepDellBootstrap.ui (hunk 1 above)
Comment (ycheng) [reply ubiquity/stepDellBootstrap.ui]: modify accordingly.

File: ubiquity/stepDellBootstrap.ui (hunk 2 above)
Comment (ycheng) [reply ubiquity/stepDellBootstrap.ui:627]: modify accordingly.

**Comment by ycheng** [comment]
@superm1 we've cleaned the flag on the disk that can test the RST patch. Mind share the method the add back the flag so I can test the patch?

**Comment by superm1** [comment]
From firmware setup you should be able to turn on Intel RST option rom. From there you should be able to format disk with rst metadata.

**Comment by ycheng** [comment]
Found and test with cases on with/without RST disks.
The Warning will show/hide depends on whether there are RST disks.
And both will be sort by disk size correctly.

I don't know how to test dhc case, but it shall works I guess.

**Pull request merged**